tests/kernel/install_ltp.pm

```diff
@@ -271,6 +274,7 @@ sub run {
     upload_logs('/boot/config-$(uname -r)', failok => 1);
 
     add_we_repo_if_available;
+
```

Comment:
Is there a point adding this empty line without further changes here?

Reply:
Yes, it's not related to the code bellow.